### ISSUE TITLE
Change base branch to a new branch "debian_sid"

### ISSUE THREAD

**Issue by Eloston** [ungoogled-software/ungoogled-chromium-debian#80]
We should change the branch that tracks upstream (Debian) be a new branch called `debian_sid` that targets Debian sid. This will better match the target Debian version for upstream's `master` branch, and makes it easier separate changes specific to stable Debian versions from testing or unstable.

The current Debian stable branch `debian_buster` should track this new `debian_sid` branch. Also, Ubuntu versions should target either a specific Debian branch or `debian_sid`, depending on when they were released and/or the versions of their packages.

**Comment by ghost** [comment]
Agreed. Perhaps it would be better to create it now and redo the work I did for buster on it so that the buster specific workaround I had to apply won't be part of it. Also it should be called debian_sid or so since this is the permanent name for Debian's bleeding edge repo.

**Comment by braewoods** [comment]
Also it should be called debian_sid or so since this is the permanent name for Debian's bleeding edge repo.

**Comment by Eloston** [comment]
@braewoods I like that idea. I'll publish a new branch called `debian_sid` that contains the same code as `debian_buster` right now (with the README modified slightly).

Feel free to publish a new PR for `debian_sid`. I think we can keep the current `debian_buster` PR and simply update it.

**Comment by ghost** [comment]
Ok. Let me know when it's ready and I can work on redoing the work so I know what can be omitted from the new branch.

**Comment by Eloston** [comment]
@braewoods Should be good to go. FYI in `debian_sid`, I removed some patches that seemed to be buster-specific, but I didn't do a thorough check to see if I removed all the buster-specific changes.

**Comment by ghost** [comment]
Ok. I plan to rebase debian_sid on clang 9 because clang 10 is still not officially released. I think it's better to use 9 for a bit longer. I intend to redo the whole work due to the changes that have been made. I need to change how I merged the last upstream changes.

**Comment by ghost** [comment]
@Eloston One thing I'd like to ask you though. The upstream added patches to "disable" whatever this third_party/catapult is but we have patches that apply under it so we may need to tweak the copyright exclusions for it. How do you think it should be handled? To get it to build before I had to remove the exclusion entirely without a replacement.

**Comment by ghost** [comment]
Ok, the update to debian_sid is ready to be reviewed. The main thing that ended up being unnecessary was the changes to re2. Once this is approved I plan to backport it to update the other branches.

**Comment by Eloston** [comment]
> The upstream added patches to "disable" whatever this third_party/catapult is but we have patches that apply under it so we may need to tweak the copyright exclusions for it. 

This is one component of browser tracing/profiling. Tracing/profiling is used in DevTools and a few other places. There was an issue about reverting Debian's choice to disable tracing/profiling either in this repo or the main repo, so we should continue to honor that request. That means we should ensure that Catapult remains enabled.

**Comment by Eloston** [comment]
Fixed Catapult with 79c78c3bcaef9c50471284aaf9ad33b880c92035
